layers/cmd/valueset.go

```diff
@@ -311,6 +321,195 @@ func LoadValuesetFiles(vs *Valuesets, files []string) error {
 	return nil
 }
 
+func (opt Options) splitCell(header, cell string) []string {
+	if sep, exists := opt.Separator[header]; !exists {
+		return []string{cell}
+	} else {
+		var ret []string
+		// if separator is empty, default to space
+		sep = strings.TrimSpace(sep)
+		if sep == "" {
+			s := strings.Split(cell, " ")
+			for _, str := range s {
+				if str != "" {
+					ret = append(ret, str)
+				}
+			}
+			return ret
+		}
+		s := strings.ReplaceAll(cell, " ", "")
```

Comment:
Trim each element individually. This will break, say :+1: 
abc def ; ghi jkl l; xyz